cluster/resolver/rpc.go

```diff
@@ -9,32 +9,34 @@
 //  CONTACT: hello@weaviate.io
 //
 
-package rpc
+package resolver
 
 import (
 	"fmt"
 	"net"
 	"strconv"
 )
 
-// NewResolver returns an implementation of rpcAddressResolver
-// isLocalHost is used to determine which remote port to expect given an address (See: rpcResolver.rpcAddressFromRAFT())
-// rpcPort is used as the default port on the returned rpcAddresses (see: rpcResolver.Address())
-func NewResolver(isLocalHost bool, rpcPort int) rpcAddressResolver {
-	return &rpcResolver{isLocalCluster: isLocalHost, rpcPort: rpcPort}
-}
-
-type rpcResolver struct {
+// RpcResolver implements resolving raft address to their RPC address depending on the configured rpcPort and whether or
+// not this is a local cluster.
+type RpcResolver struct {
 	isLocalCluster bool
 	rpcPort        int
 }
 
+// NewRpcResolver returns an implementation of rpcAddressResolver
+// isLocalHost is used to determine which remote port to expect given an address (See: rpcResolver.rpcAddressFromRAFT())
+// rpcPort is used as the default port on the returned rpcAddresses (see: rpcResolver.Address())
+func NewRpcResolver(isLocalHost bool, rpcPort int) *RpcResolver {
```

Comment:
**thought:**  shall we call it `NewRPC` given that it will be used `resolver.NewRPC()`,  wdyt?